lib/openshift/service_monitor.rb

```diff
@@ -0,0 +1,5 @@
+module BushSlicer
+  class ServiceMonitor < ProjectResource
+    RESOURCE = "servicemonitors"
```

Comment:
did you verify this is a Project resource?

Reply:
I'm not 100% certain,  this is all I have from `oc explain`

> KIND:     ServiceMonitor
> VERSION:  monitoring.coreos.com/v1
> 
> DESCRIPTION:
>      <empty>

Reply:
There is an easy way to figure out whether a resource is namespaced or not:

https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#not-all-objects-are-in-a-namespace

Reply:
`ServiceMonitor` is namespaced resource.

Reply:
@akostadinov, I'm not sure what/how to use that command mentioned in your link.  Can you please specify?

Reply:
Just run the commands and see list. Anyway, @lihongan has confirmed apparently.